frontend/src/pages/Properties/Properties.tsx

```diff
@@ -47,78 +48,80 @@ const Properties = () => {
   // Presentational layer
   return (
     <Container>
-      <HouseHeader>Recent Properties</HouseHeader>
-      <Link to='/properties/new'>
-        <Button text='New Property' />
-      </Link>
-      {loading ? '...Loading' : null}
-      {error.error
-        ? 'Whoops! There was an error loading this content for you ☹️'
-        : null}
-      {houses
-        ? houses.map((house: House) => {
-            return (
-              <HouseItem key={house.id} data-testid='house-item'>
-                <ThumbNail
-                  src='https://www.samplemcdougald.org/wp-content/uploads/2017/10/visit-sample-mcdougald-300x300.jpg'
-                  alt='house'
-                />
-                <CardContent>
-                  <CardHeading>
-                    <h4>{house.name}</h4>
-                    <p>{house.address}</p>
-                  </CardHeading>
-                  <CardBody>
-                    <CheckList>
-                      <p>Checklist Items</p>
-                      {house.checkList[0].count}
-                    </CheckList>
-                    <ButtonContainer>
-                      <Link
-                        to={{
-                          pathname: `properties/${house.id}`,
-                          hash: '#checklists',
-                          state: house,
-                        }}
-                      >
-                        <Button
-                          text='Edit Checklists'
-                          datatestid='house-button'
-                        />
-                      </Link>
-                      <Link to={`/houses/${house.id}#resources`}>
-                        <Button
-                          text='Edit Resources'
-                          datatestid='house-button'
-                        />
-                      </Link>
-                    </ButtonContainer>
-                    <Cleaner>
-                      Default Cleaner
-                      <select
-                        data-testid='assistant-select'
-                        onChange={(event) => postAst(event, house.id)}
-                      >
-                        <option defaultValue={house.default_ast_name}>
-                          {house.default_ast}: {house.default_ast_name}
-                        </option>
-                        {house.openAst.map((ast: any) => {
-                          if (ast.ast_id !== house.default_ast) {
-                            return (
-                              <option key={ast.ast_id}>
-                                {ast.ast_id}: {ast.full_name}
-                              </option>
-                            );
-                          }
-                        })}
-                      </select>
-                    </Cleaner>
-                  </CardBody>
-                </CardContent>
-              </HouseItem>
-            );
-          })
-        : null}
+			<PropContainer>
```

Comment:
I think either you aren't using prettier or it has gone rogue on you, this formatting is out of whack.

Reply:
Ok, I think it looks better now.